db/convenience.cc

```diff
@@ -27,6 +27,36 @@ Status DeleteFilesInRange(DB* db, ColumnFamilyHandle* column_family,
       ->DeleteFilesInRange(column_family, begin, end);
 }
 
+Status VerifyChecksum(const std::string& file_path) {
+  unique_ptr<RandomAccessFile> file;
+  uint64_t file_size;
+
+  Options options;
+  ImmutableCFOptions ioptions(options);
+  EnvOptions env_options;
+  InternalKeyComparator internal_comparator(BytewiseComparator());
```

Comment:
You can assume this is byte wise compactor. You should ask users to pass in the information needed. Same for env_options, Env, etc.